### ISSUE TITLE
iOS SecureEntry SDK uses hard copies of the dependencies

### ISSUE THREAD

**Issue by bpoplauschi** [ticketmaster/iOS-SecureEntrySDK#2]
Hi,

I think it would make a cleaner SDK if the hard copies of all the dependencies were removed from this repo and referenced directly from their dedicated repos.
There are a few options to achieve that:
- use simple git submodules to link to those repos - will keep most of the setup similar
- reference those dependencies using a dependency management system like CocoaPods or Carthage

What is your take on this matter?
I can help with the actual work, just let me know.

**Comment by bpoplauschi** [comment]
Another "issue" is having local modifications on those files, but that can be also cleaned up

**Comment by karlwhite** [comment]
Absolutely, I totally agree on both points... and both are planned, but were deferred initially for expediency and to get this into the hands of developers quickly. The plan is to initially add these items as submodules, but later create a CocoaPod/Carthage release of this framework, and replace these as dependencies.

I would certainly welcome any contributions! :)

**Comment by bpoplauschi** [comment]
Thanks for the quick response @karlwhite and glad to see you are open to contributions :)

I have already created #1 with CocoaPods support for the existing 1.0.1 version. You can merge that and it can be published on the CocoaPods Trunk repo - I can assist with that.

I will find some time to create another PR with cleaning up some of these dependencies.
Keep in touch.

**Comment by karlwhite** [comment]
Thanks for the PR @bpoplauschi! I have added the description and merged this request. I am not yet submitting to the CocoaPods trunk, but in the meantime have added instructions to the README.md for installing to the PodFile using this git repo url directly.

**Comment by bpoplauschi** [comment]
Thanks @karlwhite - actually now that the podspec is merged, we can already install the SDK via CocoaPods as you did in the example (using a `git` reference).
